hbw/app/assets/stylesheets/hbw/variables.sass

```diff
@@ -26,3 +26,5 @@ $btn-danger-bg: transparent
 $btn-danger-color: $btn-danger-border
 
 $deadline-red: $brand-danger
+
+$font-size-normal: 15px
```

Comment:
Let's convert this to CSS-variable like `--base-font-size` and use it in branding styles right now :)